resources/begrep/sikkerDigitalPost/xsd/sdp-manifest.xsd

```diff
@@ -6,6 +6,25 @@
 
 	<xsd:include schemaLocation="sdp-felles.xsd"/>
 
+    <xsd:complexType name="ManifestMottaker">
+        <xsd:sequence>
+            <xsd:element name="person" type="ManifestPerson" minOccurs="1" maxOccurs="1"/>
+        </xsd:sequence>
+    </xsd:complexType>
+
+    <xsd:complexType name="ManifestPerson">
+        <xsd:sequence>
+            <xsd:element name="personidentifikator" minOccurs="1" maxOccurs="1">
```

Comment:
Skulle vi ikke droppe personidentifikator i manifestet? Dvs. ManiFestPerson.personidentifikator?

Reply:
Jeg vet ikke. Hvis dere finner ut at det ikke er behov for den, så bør den nok fjernes.